### PR TITLE
doc: add simple readme for etcd backend

### PR DESCRIPTION
**Pull request by xiang90**

### CODE REVIEW

File: physical/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Physical backends
+
+### etcd Backend
+
+etcd backend supports both v2 and v3 API. To explicitly specify the API version, add `etcd_api`
+field to the backend section in the configuration file.
```
Comment (philips) [inline physical/README.md]: And the default is auto-detected based on your cluster?

File: physical/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## Physical backends
+
+### etcd Backend
+
+etcd backend supports both v2 and v3 API. To explicitly specify the API version, add `etcd_api`
```
Comment (heyitsanthony) [inline physical/README.md]: s/v2 and v3 API/v2 and v3 APIs

File: physical/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+### etcd v3 backend
+
+etcd v3 backend is maintained by [etcd team](https://github.com/coreos/etcd/blob/master/MAINTAINERS#L1-L4). It supports all backend features including HA support.
+
+etcd version 3.1+ is required to enable v3 backend.
+
+### etcd v2 backend
+
+etcd v2 backend has known issue with HA support.
```
Comment (heyitsanthony) [inline physical/README.md]: s/issue/issues

**Comment by xiang90** [comment]
/cc @raoofm

hope this helps.

**Comment by raoofm** [comment]
@xiang90 gr8 thanks much.

**Comment by jefferai** [comment]
When you proposed this you said something about in the etcd dir, I thought you had factored the code into packages. This info should just go onto the config page like the other backends. There is a long-term todo to split that into separate sub pages for each one, but we're not there yet.

**Comment by xiang90** [comment]
@jefferai I can move etcd to its own directory, and add this in. How do you feel about changing the website bit?

**Comment by jefferai** [comment]
I'm not going to have just one backend in its own section and all of the others in the current section...that's even more of a UX nightmare than the current situation.

**Comment by xiang90** [comment]
@jefferai 

I mean the 3 lines change in website/source/docs/config/index.html.md

**Comment by xiang90** [comment]
@jefferai 

Or can you give me a way to communicate that etcd3 is maintained by etcd team. So that the Vault users can easily report bugs to. Or we have to watch the Vault repo, and check through every issues everyday, which is unlikely to happen. We will miss issues, and Vault users wont have the best experience as they should.

**Comment by jefferai** [comment]
I'd just change the change in index.html.md to say "this backend is maintained by the etcd team" with the link.

**Comment by xiang90** [comment]
@jefferai OK. I will change this patch and ping you back once it is done.

**Comment by jefferai** [comment]
Migrated the info in c05d74be59d2e7c817277e37c17d03e166e5c52e

**Comment by xiang90** [comment]
@jefferai Thank you!

**Pull request closed without merging**